src/app/Authenticated.tsx

```diff
@@ -74,76 +77,86 @@ export const routeDetails = {
 };
 
 const Authenticated = () => {
-    return (
-        <Routes>
-            <Route
-                path={routeDetails.registration.path}
-                element={<Registration />}
-            />
-            <Route element={<AppLayout />}>
-                <Route path={routeDetails.home.path} element={<Home />} />
+    const { combinedGrants, isValidating } = useCombinedGrantsExt({
+        singleCall: true,
+    });
```

Comment:
Use `singleCall` settings so the call does not repeat constantly causing re-renders